api/wasm/indigo-ketcher/CMakeLists.txt

```diff
@@ -53,6 +53,23 @@ add_custom_target(${PROJECT_NAME}-test
         DEPENDS ${PROJECT_NAME}
         )
 
+if(RENDER_ENABLE_CJK)
+        add_custom_target(${PROJECT_NAME}-test-cjk
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/test_cjk.js ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/
```

Comment:
why do we copy tests inside cmake build?

Reply:
This file is copied to run tests inside "build" folder.
The same way as is it done for no CJK tests.

Reply:
can we remote hardcoded tests name from the cmake?

Reply:
yes, done